stdlib/LinearAlgebra/src/symmetric.jl

```diff
@@ -920,9 +920,9 @@ function ^(A::Hermitian{T}, p::Real) where T
     end
 end
 
-for func in (:exp, :cos, :sin, :tan, :cosh, :sinh, :tanh, :atan, :asinh, :atanh)
+for func in (:exp, :cis, :cos, :sin, :tan, :cosh, :sinh, :tanh, :atan, :asinh, :atanh)
     @eval begin
-        function ($func)(A::HermOrSym{<:Real})
+        function ($func)(A::Union{HermOrSym{<:Real},SymTridiagonal{<:Real}})
```

Comment:
This is an unrelated change, so it would be better to make this a separate PR.

Reply:
No problem, I'll open a separate PR for `SymTridiagonal`.